browser/src/map/Clipboard.js

```diff
@@ -536,33 +530,31 @@ L.Clipboard = L.Class.extend({
 	populateClipboard: function(ev) {
 		this._checkSelection();
 
-		if (this._navigatorClipboardWrite()) {
-			// This is the codepath where the browser initiates the clipboard operation,
```

Comment:
I found this comment useful: there are two cases to handle (user clicks on copy button or user presses Ctrl-C) and it's not trivial which one is handled where. Would you consider restoring it?

Also, the early return had the benefit of keeping the intent level small. A usual refactor is to "flatten" a function by adding such early returns. Is there a benefit to do the opposite of that here?

Reply:
Restored :)